src/main/java/org/testcontainers/containers/TarantoolContainer.java

```diff
@@ -244,15 +246,9 @@ protected void configure() {
             throw new IllegalArgumentException(
                     String.format("No resource path found for the specified resource %s", getDirectoryBinding()));
         }
+        String sourceDirectoryPath = normalizeBindingPath(sourceDirectory.getPath());
 
-        String serverScriptPath = Paths.get(getDirectoryBinding(), getScriptFileName()).toString();
```

Comment:
The behavior has changed -- the script is not checked for existence anymore. Is this an intended change?

Reply:
Reverted and refactored to explicit method call for better readabilty.